demo/admin/src/products/future/generated/ProductsGrid.tsx

```diff
@@ -124,6 +124,15 @@ export function ProductsGrid({ filter, toolbarAction, rowAction }: Props): React
                 { value: "Shirt", label: intl.formatMessage({ id: "product.type.shirt", defaultMessage: "Shirt" }) },
                 { value: "Tie", label: intl.formatMessage({ id: "product.type.tie", defaultMessage: "Tie" }) },
             ],
+            renderCell: ({ row }) => {
+                const valueOptions = [
+                    { value: "Cap", label: intl.formatMessage({ id: "product.type.cap", defaultMessage: "great Cap" }) },
+                    { value: "Shirt", label: intl.formatMessage({ id: "product.type.shirt", defaultMessage: "Shirt" }) },
+                    { value: "Tie", label: intl.formatMessage({ id: "product.type.tie", defaultMessage: "Tie" }) },
+                ];
```

Comment:
You are defining the valueOptions again, that are already defined in the ColumnConfig. There must be a way to access the column config to avoid that.

(not a big problem for generated code, but if you use that code for scaffolding it is)

Reply:
The `valueOptions` inside `renderCell` is removed in the follow-up PR: https://github.com/vivid-planet/comet/pull/2422

Reply:
However, you're still defining all options twice. I believe @nsams meant that we should get rid of that duplication

Reply:
I would like to have waited for https://github.com/vivid-planet/comet/pull/2422#discussion_r1724624780 to be resolved in the follow-up PR to avoid conflicts, but I just realized, the `valueOptions` are easily accessible from the params anyway 😄 

I removed the duplication in fcadca5fff080ee1043c50ef977a725b17325736